cmd/thanos/compact.go

```diff
@@ -457,6 +460,19 @@ func runCompact(
 
 		return cleanPartialMarked()
 	}
+	g.Add(func() error {
+		ps := compact.NewDefaultPlanSim(grouper, planner, sy)
+		numberOfIterations, numberOfBlocksMerged, err := ps.Simulate(context.Background())
+		if err != nil {
+			return errors.Wrapf(err, "could not simulate planning")
+		}
+		level.Info(logger).Log("msg", "number of iterations performed", numberOfIterations)
+		level.Info(logger).Log("msg", "number of blocks merged", numberOfBlocksMerged)
```

Comment:
This usage is incorrect. Each value requires a key. So here `msg="number of blocks merged"` and you need to add a key for `numberOfBlocksMerged` as well.
Also, let's make log level to debug.

Reply:
Sorry for the oversight. Fixed now.